requirements.txt

```diff
@@ -1,4 +1,4 @@
-CoolProp
+coolprop
```

Comment:
On PyPi the package is title case, but all the other packages listed out in the reqs file are usually lower case, so this is just an impulse thing.  Feel free to thumbs down it and I'll revert it.

Reply:
👍